cmd/load.go

```diff
@@ -0,0 +1,366 @@
+// Copyright 2022 The NATS Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"net/url"
+	"os/exec"
+	"strings"
+
+	"github.com/nats-io/jwt/v2"
+	"github.com/nats-io/nkeys"
+	"github.com/nats-io/nsc/cmd/store"
+	"github.com/nats-io/nsc/home"
+	"github.com/spf13/cobra"
+)
+
+func createLoadCmd() *cobra.Command {
+	var params LoadParams
+	cmd := &cobra.Command{
+		Use:   "load",
+		Short: "install entities for an operator, account and key",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// Note: Need to initialize the operator and environment
+			// before the RunAction command since that is dependent
+			// on an operator being available.
+			params.setupHomeDir()
+			params.addOperator()
+			err := RunAction(cmd, args, &params)
+			if err != nil {
+				switch err.Error() {
+				case "operator not found":
+					cmd.Printf("Unable to find operator %q\n", params.operatorName)
```

Comment:
should the message instead tell them to select the correct store or define the operator name in the environment as `NSC_<OPERATOR>_OPERATOR http/s://host:port/jwt/v1/operator`....

Reply:
sounds good to me

Reply:
Updated error message:

```
Unable to find operator "KOo"

If you have used this operator, please enter:`nsc env -s /path/to/storedir`
or define the operator endpoint environment:
`NSC_<OPERATOR_NAME>_OPERATOR=http/s://host:port/jwt/v1/operator`

Error: operator not found
```